revoke/revoke_test.go

```diff
@@ -219,6 +263,39 @@ func TestRemoteFetchError(t *testing.T) {
 
 }
 
+func TestLocalCRL(t *testing.T) {
+	crl, err := x509.ParseCRL([]byte(testCRL))
+	if err != nil {
+		panic(err.Error())
+	}
+	defaultChecker.localCRL.path = "/RootSignPartners.crl"
```

Comment:
again this path is a invalid path, but because crl is directly assigned, the test below works, this is another bad example of testing.